ui/core/src/model/time.ts

```diff
@@ -31,12 +32,26 @@ export interface RelativeTimeRange {
 export type TimeRangeValue = AbsoluteTimeRange | RelativeTimeRange;
 
 /**
- * Determine whether a given time range is relative or absolute
+ * Determine whether a given time range is relative
  */
-export function isRelativeValue(timeRange: TimeRangeValue): timeRange is RelativeTimeRange {
+export function isRelativeTimeRange(timeRange: TimeRangeValue): timeRange is RelativeTimeRange {
   return (timeRange as RelativeTimeRange).pastDuration !== undefined;
 }
 
+/**
+ * Determine whether a given time range is in Grafana format
+ */
+export function isGrafanaRelativeTimeRange(from: string, to: string): boolean {
```

Comment:
`isGrafana` ? :D. Maybe another name would be more appropriate don't you think ?

Reply:
Are we trying to avoid using 'Grafana' in our codebase? For me, the `from: 'now-<duration>', to: 'now'` structure is kind of specific to Grafana and I needed some way to distinguish it from the isRelativeTimeRange util

Reply:
well from my side, it was weird to the `Grafana` named here. It's really a typed :).

if `to` is always equal to `now` in that method, why not rename it with something like `TimeFromPastToNow` (not super english though)

Reply:
That makes sense, this util actually isn't needed anymore based on the other thread so ended up removing it completely!